decidim-proposals/config/locales/en.yml

```diff
@@ -494,6 +495,8 @@ en:
           deleted_proposals_info: Deleted proposals can be restored from the trash.
           preview: Preview
           view_deleted_proposals: View deleted proposals
+        deprecation_warning_participatory_texts_html: "⚠️ <b>Deprecation Notice.</b> The Participatory Texts feature will be removed in Decidim v0.32. Organizations using this feature should plan to migrate to the \"decidim-collaborative_texts\" module."
```

Comment:
Remove this deprecation notice since we're not deprecating Participatory Texts yet.

Reply:
Ok!